o1vm/src/legacy/main.rs

```diff
@@ -159,7 +159,7 @@ pub fn main() -> ExitCode {
         for i in 0..N_MIPS_REL_COLS {
             match i.cmp(&SCRATCH_SIZE) {
                 Ordering::Less => mips_trace.trace.get_mut(&instr).unwrap().witness.cols[i]
-                    .push(mips_wit_env.scratch_state[i]),
+                    .push(mips_wit_env.scratch_state[i].to_field()),
```

Comment:
Basically here, it looks incremental to me, as opposed to batch inverses. Let's wait for some benchmarks perhaps, but my intuition says it will be similar to what we currently have but just more intricate 🤔

Reply:
Indeed it's not bacthed here, but we do not maintain the legacy folder (or should we ?)

Reply:
Ah I see, I didn't realize of the directories